doc/lua_api.md

```diff
@@ -9097,11 +9097,13 @@ Player properties need to be saved manually.
     hp_max = 10,
     -- Defines the maximum and default HP of the entity
     -- For Lua entities the maximum is not enforced.
-    -- For players this defaults to `minetest.PLAYER_MAX_HP_DEFAULT`.
+    -- For players this defaults to `core.PLAYER_MAX_HP_DEFAULT` (20).
+    -- `core.PLAYER_MAX_HP_DEFAULT` is defined in `builtin/game/constants.lua`
```

Comment:
as already said, this is not necessary, so imho is best if removed (same down below)